conf/messages

```diff
@@ -148,8 +151,7 @@ undertaking.confirmation.p4=If you have any other businesses, you must add them
 
 businessEntityEori.eoriInUse.error=This EORI is in use.
 
-#account.title is temporary and can be changed later, as test were failing because of missing account.title
-account.title = {0}
+account-homepage.title = {0}
```

Comment:
If you are changig this, then you need to change test case where I have used #account.title